WordPress/Sniffs/PHP/YodaConditionsSniff.php

```diff
@@ -46,6 +46,8 @@ public function process_token( $stackPtr ) {
 		$beginners   = PHP_CodeSniffer_Tokens::$booleanOperators;
 		$beginners[] = T_IF;
 		$beginners[] = T_ELSEIF;
+		$beginners[] = T_RETURN;
+		$beginners = array_merge( $beginners, PHP_CodeSniffer_Tokens::$assignmentTokens );
```

Comment:
Definitely! This list could be created & assigned to a property in the `register()` method, similar like we already do in several other sniffs.